app/views/tag/index.html.erb

```diff
@@ -1,4 +1,7 @@
 <div class="col-md-12">
+  <div class="alert alert-success" id="follow-success" style="display: none;position: absolute;">
```

Comment:
Here, maybe we need to use Noty to display something using pure JavaScript, so we can consolidate all the JS for this into one place, rather than inserting new HTML... you can read about Noty here:

https://github.com/publiclab/plots2/search?utf8=%E2%9C%93&q=noty&type=

Reply:
Okay, that looks good I will change that!